src/project/serve/serve.ts

```diff
@@ -554,6 +554,13 @@ export async function serveProject(
   }
 }
 
+// https://deno.com/blog/v1.23#remove-unstable-denosleepsync-api
```

Comment:
It looks like you got things to work w/ Deno 1.23 w/ this change. That's fine, but as an FYI there are more changes required for 1.24 (and we currently have an issue whereby 1.24 crashes on M1 macs so we can't update to it).

Reply:
Is there an issue you can point me to about this M1 problem? I'd like to test my packages against it, and get it working if I can.

Reply:
This is a PR that has the upgrade to 1.24.1: https://github.com/quarto-dev/quarto-cli/pull/1945

Everything works great except if you render on an newer M1 Mac you see sporadic crashing (at the Rust runtime level). We speculated that it could be FFI issues however we disabled deno dom native and still saw the crash (so perhaps in WASM loading which we do in a number of places).

Reply:
Note that we don't plan on trying to merge this until v1.2 so your current PR should stay based on Deno 1.23 if want this to land in v1.1